## Detection.msg
Header header

string object_class

# targe number
uint32 num

# probability
float32 p

# size of bounding box
uint16 x
uint16 y

uint16 width
uint16 height